src/app/components/hero-action/hero-action.component.html

```diff
@@ -1,3 +1,2 @@
-<div>
```

Comment:
If you are making a mock component next time please tell it so in the file name too

Reply:
came from master, only modified the contents